tests/unit/features/partitions/executor/test_migration.py

```diff
@@ -23,6 +23,9 @@ class TestFileMigration(test_migration.TestFileMigration):
     """Tests for file migration with partitions enabled."""
 
 
+@pytest.mark.xfail(
+    reason="fails to create envvars for namespaced partitions, will be fixed by #597"
+)
```

Comment:
Oh nice catch!

I'll leave it unstrict since some tests in these imported test suites pass.

And like you said, they are going to be removed in the next PR.